src/Tizen.TV.UIControls.Forms/Handler/TVEntryHandler.cs

```diff
@@ -1,58 +1,50 @@
-// * limitations under the License.
-// */
-//using System;
-//using Microsoft.Maui.Controls;
-//using Microsoft.Maui.Handlers;
+﻿/*
+ * Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 
-//namespace Tizen.TV.UIControls.Forms.Handler
-//{
-//    public class TVEntryHandler : EntryHandler
-//    {
-//        const string _doneKeyName = "Select";
-//        const string _cancelKeyName = "Cancel";
+using System;
+using Microsoft.Maui.Controls;
+using Microsoft.Maui.Handlers;
+using TEntry = Tizen.UIExtensions.NUI.Entry;
 
-//        protected override void ConnectHandler(UIExtensions.ElmSharp.Entry platformView)
-//        {
-//            base.ConnectHandler(platformView);
+namespace Tizen.TV.UIControls.Forms.Handler
+{
+    public class TVEntryHandler : EntryHandler
+    {
+        const string _doneKeyName = "Select";
 
-//            if (platformView != null)
-//            {
-//                InputEvents.GetEventHandlers(VirtualView as BindableObject)?.Add(new RemoteKeyHandler(new Action<RemoteControlKeyEventArgs>((args) =>
-//                {
-//                    if (args.PlatformKeyName.Equals(_doneKeyName))
-//                    {
-//                        //FocusSearch(true)?.SetFocus(true);
-//                        Device.BeginInvokeOnMainThread(() =>
-//                        {
-//                            VirtualView.Text = platformView.Text;
-//                            (VirtualView as IEntryController)?.SendCompleted();
-//                        });
-//                    }
-//                    else if (args.PlatformKeyName.Equals(_cancelKeyName))
-//                    {
-//                        platformView.HideInputPanel();
-//                    }
-//                }), RemoteControlKeyTypes.KeyDown));
+        protected override void ConnectHandler(TEntry platformView)
+        {
+            base.ConnectHandler(platformView);
 
-//                if (platformView is Tizen.UIExtensions.ElmSharp.Entry entry)
-//                {
-//                    entry.EntryLayoutFocused += OnFocused;
-//                    entry.EntryLayoutUnfocused += OnUnfocused;
-//                }
-//            }
-//        }
-//    }
-//}
+            if (platformView != null)
+            {
+                InputEvents.GetEventHandlers(VirtualView as BindableObject)?.Add(new RemoteKeyHandler(new Action<RemoteControlKeyEventArgs>((args) =>
```

Comment:
These code to handle key event in `TVEntryHandler` and `TVEditorHandler` look exactly same. how about you extract a method or combine two handlers? I think that it can improve code readability getring rid of duplicated code.

Reply:
Oh right, that sounds really great in the clean code perspective. It looks better to make a common logic for this. Just one thing is now `TVEntryHandler` and `TVEditorHandler` are separated, so I would need to make another class to cover all.
I would, next time, figure out how I can combine for the common logic and apply!